HLTrigger/Configuration/test/sonic_hlt_test.py

```diff
@@ -0,0 +1,32 @@
+import FWCore.ParameterSet.Config as cms
+import os,sys
+
+sys.path = sys.path + [os.path.expandvars("$CMSSW_BASE/src/HLTrigger/Configuration/test/"), os.path.expandvars("$CMSSW_RELEASE_BASE/src/HLTrigger/Configuration/test/")]
+
+from OnLine_HLT_GRun import process
+
+process.hltHbherecopre = process.hltHbhereco.clone(
+    makeRecHits = cms.bool(False),
+    saveInfos = cms.bool(True),
+)
+
+process.hltHbhereco = cms.EDProducer("HcalReconstructor",
+    Client = cms.PSet(
+        batchSize = cms.untracked.uint32(16000),
+        address = cms.untracked.string("ailab01.fnal.gov"),
```

Comment:
probably better to make the default server `0.0.0.0`, so we don't have a ton of random people pinging ailab